### PR TITLE
Round pay amounts in distribution modal

### PR DESCRIPTION
**Pull request by NakedFool**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/2784

I've rounded amounts to the precision of 4 since that is how we round `grossAvailableAmount` that is presented under input.

Some smol additions that I've included in this PR:

- I've added a tooltip for text under input '{amount} available to distribute'
- I've added also a tooltip for the fee amount, so now the user can see what is fee amount in USD when it is about to distribute.

I'll attach a video so you can see it all in action.

### Before
image

### After
image

https://user-images.githubusercontent.com/18723426/213519302-a02a4643-7846-4063-ab93-b51cb3a26efb.mov

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@NakedFool this looks good to me. I think our strategy in general should be

ETH => >4 decimal places 
USD => 2 decimal places

Be good to get consistent on this across the board!

**Review by tomquirk — APPROVED** [review]

File: src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx [hunk 1]
```diff
@@ -151,7 +159,7 @@ export default function DistributePayoutsModal({
           >
             <FormattedNumberInput
               placeholder="0"
-              value={distributionAmount}
+              value={grossAvailableAmount}
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx:162]: **issue** I havent tested, but just from reading the code, this looks like a potential issue. `grossAvailableAmount` isn't set by the user (or influenced by any mutable value); `distributionAmount` is (via `setDistributionAmount`).

Should we be changing this?

**Review by tomquirk — CHANGES_REQUESTED** [review]

File: src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx (hunk 1 above)
Comment (NakedFool) [reply src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx]: You are right, I'm sorry. 

I guess that I've misread distributable with distributionAmount, things should be cool now.I've just added precision to max button and in starting hook.

**Comment by NakedFool** [comment]
> @NakedFool this looks good to me. I think our strategy in general should be
> 
> ETH => >4 decimal places USD => 2 decimal places
> 
> Be good to get consistent on this across the board!


Sure! I'll keep it in mind moving forward.

**Review by tomquirk — APPROVED** [review]

**Pull request merged**